libbeat/publisher/pipeline/ttl_batch.go

```diff
@@ -93,10 +93,14 @@ func (b *ttlBatch) Events() []publisher.Event {
 }
 
 func (b *ttlBatch) ACK() {
+	// Help the garbage collector clean up the event data a little faster
+	b.events = nil
```

Comment:
Should this (and the one below) be done after calling `b.done()` in case the implementation of `done` references the contents of `events` some time in the future?

Reply:
The done callback is just to forward acknowledgments (and most of that is async, so changing the order wouldn't help), but it has no access to the contents of the batch